Countries/CountryTableViewCell.swift

```diff
@@ -96,15 +93,15 @@ class CountryTableViewCell: UITableViewCell {
     
     override func layoutSubviews() {
         super.layoutSubviews()
-        gradient.frame = mainView.bounds
+        gradient.frame = containerView.bounds
```

Comment:
any other place to set this? and is it necessary?

Reply:
yes, there probably is, but I didn't find it..
I need to search more